tools/wltests/test_series

```diff
@@ -385,8 +377,28 @@ energy_measurement:
     # set them here:
     iio_devices: [$IIO_DEVICES]
 EOF
+}
+
+# Prepare WA_AGENDA (if specified it override the TEST_CMD)
+if [ ! -z $WA_AGENDA ]; then
+	if [ ! -f $WA_AGENDA ]; then
+		echo
+		c_error "The \$WA_AGENDA|--wa_agenda is not a valid path for a WA agenda"
+		list_available_agendas
+		exit $INVAL
+	fi
 	TEST_CMD_RESULTS="$RESULTS/wa.\${COMMIT_SHA1:0:7}_\${COMMIT_NAME}"
-	TEST_CMD="wa run -f -d \"$TEST_CMD_RESULTS\" -c \"$WA_LOCAL_CONF\" \"$WA_AGENDA\""
+
+	# Configure WA command
+	# Depending on the EnergyMeter in use, an auxiliary configuration file
+	# can be required.
+	if [ "x$EMETER" == "xACME" ]; then
+		config_emeter_acme
+		TEST_CMD="wa run -f -d \"$TEST_CMD_RESULTS\" -c \"$WA_LOCAL_CONF\" \"$WA_AGENDA\""
+	fi
+	if [ "x$EMETER" == "xMONSOON" ]; then
```

Comment:
I'm not against this per se but I would prefer to have the support for Monsoon in a different PR.

Reply:
It's still conceptually Pixel2 support, since we well know what Google uses for EM.
Thus, considering also that's just one single patch, if you are not strongly against it, I would keep it here.

Reply:
The issue is, I cannot test it since I don't have a monsoon so I won't be able to merge this PR before someone else tests it.

Reply:
Chris can test it... or we can borrow Chri's monsoon.... but yes, lemme try removing this patch from this series. ;-)

Reply:
Pushed a new version just to add a WA fix missing... before trying to remote the MONSOON support.